# Copyright (c) 2024 Qualcomm Innovation Center, Inc. All rights reserved.
# SPDX-License-Identifier: BSD-3-Clause-Clear

# memory information
# provided by `cat /proc/meminfo`
# unit: kB
uint64 mem_total
uint64 mem_free
uint64 mem_available
uint64 buffers
uint64 cached
uint64 swap_total
uint64 swap_free
